Source/WarpX.H

```diff
@@ -1435,6 +1476,12 @@ private:
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Efield_avg_fp;
     amrex::Vector<std::array< std::unique_ptr<amrex::MultiFab>, 3 > > Bfield_avg_fp;
 
+    // Implicit, fields at start of step and from the previous iteration
```

Comment:
Could you add more comments to indicate which field is which?
(e.g. is `Efield_n` the one at the start of the step, or the one from the previous iteration)

Reply:
Comments have been added.